src/internal/cache-client.ts

```diff
@@ -152,6 +154,43 @@ export class CacheClient {
     });
   }
 
+  public async setFetch(
+    cacheName: string,
+    setName: string
+  ): Promise<SetFetch.Response> {
+    validateCacheName(cacheName);
+    validateSetName(setName);
+    return await this.sendSetFetch(cacheName, this.convert(setName));
+  }
+
+  private async sendSetFetch(
+    cacheName: string,
+    setName: Uint8Array
```

Comment:
I thought we decided that we only allow string for `setName`?

Reply:
We did, but the protobuf takes bytes. SetFetch takes a string and passes bytes to sendSetFetch to pass to the request.

Reply:
Ah I mistook the above `setFetch` as a client-facing API. Sorry.
Got it!